build.sbt

```diff
@@ -116,6 +116,7 @@ lazy val kotlinGenerator = project
 lazy val postmanGenerator = project
   .in(file("postman-generator"))
   .dependsOn(lib, lib % "test->test")
+  .dependsOn(generated, generated % "test->test")
```

Comment:
seems to be not needed.
lib already has `  .dependsOn(generated % "compile; test->test")`